example_grpc/goclient/src/katranc/katranc/katranc.go

```diff
@@ -352,6 +354,41 @@ func (kc *KatranClient) List(addr string, proto int) {
 	}
 }
 
+func (kc *KatranClient) RealsOfVip(vip *lb_katran.Vip) *lb_katran.VipReals {
+	d := lb_katran.VipReals{}
+	reals := kc.GetRealsForVip(vip)
+	proto := ""
+	if vip.Protocol == IPPROTO_TCP {
+		proto = "tcp"
+	} else {
+		proto = "udp"
+	}
```

Comment:
Instead of these two methods we can refactor 'ListVipAndReals(..)' method above to have same common method to obtain list of vips and reals, and have two different ways of outputting - via print in the stderr (ListVipAndReals) or as json to a file (nitpick: I'd suggest a more descriptive name than just 'Services').

Reply:
I mix both function, thank you for your tips